internal/testutil/pkgdata.go

```diff
@@ -197,3 +197,16 @@ func Lnk(mode int64, path, target string) TarEntry {
 		},
 	}
 }
+
+// Hln is a shortcut for creating a hard link TarEntry structure (with
+// tar.Typeflag set to tar.TypeLink). Hln stands for "Hard LiNk".
+func Hln(mode int64, path, target string) TarEntry {
```

Comment:
Forgive me for not saying this earlier but can we rename this to `Hlk` and also rename the comment above? For some reason I think `Hlk` is read more easily as `hardlink` than `Hln`. If you disagree please tell me.